nestjs/src/courses/team-distribution/dto/team-distribution-student.dto.ts

```diff
@@ -6,7 +6,7 @@ import { PaginationMetaDto } from 'src/core/paginate/dto/Paginate.dto';
 export class TeamDistributionStudentDto {
   constructor(student: Student) {
     this.id = student.id;
-    this.fullName = `${student.user.firstName} ${student.user.lastName}`;
+    this.fullName = `${student.user.firstName ?? ''} ${student.user.lastName ?? ''}`;
```

Comment:
PersonDto has `getName` method. You can try to use it instead

Reply:
fixed